plugins/ufm_log_analyzer_plugin/src/loganalyze/pdf_creator.py

```diff
@@ -60,7 +60,8 @@ def add_images(self):
     def add_text(self):
         self.set_font("Arial", "", 12)
         output = StringIO()
-        print(self._fabric_stats_list, file=output)
+        for txt in self._txt_list:
+            print(txt, file=output)
```

Comment:
If you touched it :) (Which is great!)
Why do we need this logic? Cant we add these texts and combine them with the text we send in add_list_of_dicts_as_text ? 
(will also mean a small refactor, but it makes sense)
Not a must right now...